.gitignore

```diff
@@ -9,3 +9,5 @@
 /tmp/
 footprint.sql
 footprint.:memory:.sql
+.rbenv-gemsets
+.gems
```

Comment:
I think these two things might be specific to your tool choices, too. Or, do you think other devs will be installing gems in `.gems` too?